### PR TITLE
Feature/extend account details component so its reused on account creation flow

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- Extend `account-details` component so its reused on account creation flow
- add new field/s to `account-details` component permissions on account creation flow

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -86,6 +86,12 @@ const routes: Routes = [
       title: 'Bulk Upload',
     },
   },
+  {
+    path: 'create-account',
+    loadChildren: '@features/create-account/create-account.module#CreateAccountModule',
+    // canActivate: [AuthGuard],
```
Comment (ChazUK) [inline src/app/app-routing.module.ts]: Do we need to uncomment this

File: src/app/features/create-account/create-account-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CreateAccountComponent } from '@features/create-account/create-account/create-account.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: CreateAccountComponent,
+    data: { title: 'Home' },
```
Comment (ChazUK) [inline src/app/features/create-account/create-account-routing.module.ts]: I think this title can be removed, you have a title on the lazy loaded route in app-routing

**Comment by omarCreativeDev** [comment]
[(10) User accounts - Create User Accounts](https://trello.com/c/rCjsbkOX/137-10-user-accounts-create-user-accounts)

File: src/app/features/create-account/create-account/create-account.component.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          </span>
+          <div class="govuk-radios govuk-radios--conditional">
+            <div class="govuk-radios__item">
+              <input
+                class="govuk-radios__input"
+                [class.govuk-input--error]="submitted && form.get('permissions').errors"
+                [formControlName]="'permissions'"
+                id="permissions-conditional-1"
+                name="permissions"
+                value="edit"
```
Comment (ChazUK) [inline src/app/features/create-account/create-account/create-account.component.html]: is there an enum for this already?

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/app-routing.module.ts]: done

File: src/app/features/create-account/create-account-routing.module.ts (hunk 2 above)
Comment (omarCreativeDev) [reply src/app/features/create-account/create-account-routing.module.ts]: done

File: src/app/features/create-account/create-account/create-account.component.html (hunk 3 above)
Comment (omarCreativeDev) [reply src/app/features/create-account/create-account/create-account.component.html]: not yet, will create one once BE work is done and hook up this screen to the BE

**Pull request merged**